apps/docs/content/components/River/index.mdx

```diff
@@ -103,13 +104,15 @@ A single river component acts as a section and it should not include any child s
   </Dont>
 </DoDontContainer>
 
-### Breakout river
+### River Breakout
 
 A river breakout component has an image that fills the width of the container while providing more space for content such as text and lists.
 
-This is useful when you want to create a highlight which can be used to create more emphasis or simply break a monotonous vertical flow of left/right rivers. It can also be helpful on smaller viewports to avoid stacking two images or two chunks of text from adjacent river components.
+This is useful when you want to create a highlight which can be used to create more emphasis or simply break a monotonous vertical flow of left/right rivers. It can also be helpful on smaller viewports to avoid stacking two images or two chunks of text from adjacent river components. 
+
+As it's name suggest, a river breakout should not be the first in the stack, it should be in-between two left/right rivers.
```

Comment:
```suggestion
As its name suggest, a river breakout should not be the first in the stack, it should be positioned in-between two standard rivers.
```

On narrow viewports, there is no left/right, so suggesting that we tweak wording to reference it as the default / standard one.